### DISCUSSION TITLE
Organization issue templates open against .github repo on IOS

### DISCUSSION THREAD

**Discussion by asford** [community/community#32087] [Mobile]
Issues opened on IOS using org-level issue templates open against the <org>/.github repo, not the source repo.

This repros in the conda-forge organization. Eg:

https://github.com/conda-forge/.github/issues/25

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.77.0 (76215101)
Device: iPhone 12 (iOS 15.5)
</details>

**Comment by mxie** [comment]
Thanks for letting us know @asford! Unfortunately, I was not able to reproduce this. For my setup, I had:

- a public `org/.github` repo with an issue template
- a public `org/public-repo` repo that listed the org-level issue template as an option for new issues
- a private `org/private-repo` repo that listed the org-level issue template as an option for new issues

For both the public and private repos, I was able to successfully create an issue in their respective places and _not_ in the `.github` repo.

Are you experiencing this consistently? Is there any way that you could set up some test repos to reproduce this behavior? Any screenshots or recordings would be helpful here. Thanks!

**Comment by mfansler** [comment]
Just want to reiterate this is still an issue on v1.85.0, iOS 15.6.1. Same situation with the **conda-forge** organization: using [this issue template](https://github.com/conda-forge/.github/blob/main/.github/ISSUE_TEMPLATE/2-bot-commands.yml) on a "feedstock" repository (specifically, [`r-xtensor-feedstock`](https://github.com/conda-forge/r-xtensor-feedstock)) resulted in creating [this issue](https://github.com/conda-forge/.github/issues/27) on the `conda-forge/.github` repository rather than the repository where the issue was started.
